src/compiler/checker.ts

```diff
@@ -7301,15 +7306,6 @@ namespace ts {
 
             let targetAttributesType = getJsxElementAttributesType(node);
 
-            if (getNodeLinks(node).jsxFlags & JsxFlags.ClassElement) {
-                if (node.tagName.kind === SyntaxKind.Identifier) {
-                    checkIdentifier(<Identifier>node.tagName);
-                }
-                else {
-                    checkQualifiedName(<QualifiedName>node.tagName);
-                }
-            }

```

Comment:
Yes (minus marking things as referenced in the `checkIdentifier` case). This block was causing duplicate errors once we moved off `getResolvedSymbol` in the other function.